method/orb/util/concurrent/rollingcounter/rollingcounter.go

```diff
@@ -8,13 +8,18 @@ package rollingcounter
 
 import (
 	"crypto/rand"
+	"io"
+	"log"
 	"math/big"
 	"sync/atomic"
-
-	"github.com/hyperledger/aries-framework-go/component/log"
 )
 
-var logger = log.New("aries-framework-ext/vdr/orb") //nolint: gochecknoglobals
+var debugLogger = log.New(io.Discard, " [did-go/method/orb] ", log.Ldate|log.Ltime|log.LUTC)
+
+// SetDebugOutput used to set output of debug logs.
```

Comment:
```suggestion
// SetDebugOutput is used to set output of debug logs.
```

(same applies to the other places this comment appears